src/transformation.js

```diff
@@ -18,14 +19,14 @@ import {
   snakeCase
 } from './util';
 
-import {
-  Param,
-  ArrayParam,
-  LayerParam,
-  RangeParam,
-  RawParam,
-  TransformationParam
-} from "./parameters";
+import Layer from "./layer/layer";
+import Param from "./parameters/param";
+import ArrayParam from "./parameters/arrayparam";
+import RangeParam from "./parameters/rangeparam";
+import RawParam from "./parameters/rawparam";
+import LayerParam from "./parameters/layerparam";
+// eslint-disable-next-line import/no-cycle
```

Comment:
why do we need this here?

Reply:
TransformationParam depends on Transformation. It's not a problem in runtime, but eslint doesn't like it.